client_sdk/go/fpc/platform/params.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package platform
```

Comment:
Do we need a new package?  In particular, right now it seems purely attestation specific, so platform is not an intuitive name?  In particular as you abstract attestation, i.e., it's not that you define a SGX-EPID platform or alike?   This question mainly arose as i saw all these public functions below which i felt the developer wouldn't really have a need to see. But with a separate package i guess you had no choice ...

Reply:
ok platform is not ideal. I moved the attestation stuff to a `SGX` package.